__init__.py

```diff
@@ -1,55 +1,56 @@
 import time
 import json
-from os.path import dirname
-from adapt.intent import IntentBuilder
-from mycroft.skills.core import MycroftSkill
-from mycroft.util.log import getLogger
+from os.path import dirname, isfile
+
 import paho.mqtt.client as mqtt
-from mycroft.util import play_audio_file, resolve_resource_file
+
+from adapt.intent import IntentBuilder
+from mycroft.skills import intent_handler
+from ovos_workshop.skills import MycroftSkill
+from mycroft.util import play_audio_file
+from ovos_utils import classproperty
+from ovos_utils.process_utils import RuntimeRequirements
+from ovos_utils.log import LOG
+
 __author__ = 'aussieW (jamiehoward430) modified by JoergZ2'
-#test
-LOGGER = getLogger(__name__)
+
+
 class MyTasmotaDe(MycroftSkill):
 
     def __init__(self):
-        super(MyTasmotaDe, self).__init__(name="MyTasmotaDe")
+        super(MyTasmotaDe, self).__init__()
         self.default_location = ""
         self.protocol = "mqtt"
         self.mqttssl = ""
         self.mqttca = ""
         self.mqtthost = ""
         self.mqttport = ""
-        #self.mqttauth = self.config[""]
-        #self.mqttuser = self.config[""]
-        #self.mqttpass = self.config[""]
-
-#        if (self.protocol == "mqtt"):
-#            self.mqttc = mqtt.Client("MycroftAI")
-#            if (self.mqttauth == "yes"):
-#                self.mqttc.username_pw_set(self.mqttuser,self.mqttpass)
-#            if (self.mqttssl == "yes"):
-#                self.mqttc.tls_set(self.mqttca)
-#            LOGGER.info("AJW - connect to: " + self.mqtthost)
-#            LOGGER.info("AJW - connect to: " + str(self.mqttport))
-#            self.mqttc.connect(self.mqtthost,self.mqttport,10)
-#            self.mqttc.on_message = self.on_message
-#            self.mqttc.subscribe(dataRequestTopic)
-#            self.mqttc.loop_start()
-
+    
+    @classproperty
+    def runtime_requirements(self):
+        return RuntimeRequirements(internet_before_load=False,
+                                   network_before_load=True,
+                                   gui_before_load=False,
+                                   requires_internet=False,
+                                   requires_network=True,
+                                   requires_gui=False,
+                                   no_internet_fallback=False,
+                                   no_network_fallback=False,
+                                   no_gui_fallback=False)
```

Comment:
technically `no_gui_fallback` should be True if `requires_gui=False`, should not have any impact in code but that's what we have been doing in other skills, if something downstream is doing a check against this and ignoring `required_gui` that safeguards that check. But i guess this should be handled in whatever place is checking runtime_requirements not up to the skill. so just a nitpick comment